test/functional/testnvim.lua

```diff
@@ -642,7 +642,7 @@ function M.source(code)
 end
 
 function M.has_powershell()
-  return M.eval('executable("' .. (is_os('win') and 'powershell' or 'pwsh') .. '")') == 1
```

Comment:
I think this check can be removed, because in the powershell v7.4 (LTS) the executable was renamed to `pwsh.exe` [see in the docs](https://learn.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell?view=powershell-7.4#powershell-executable-changes)

Reply:
Question would be also, if we want to support `Windows PowerShell 5.1` which is far behind the cross platform powershell version and I guess our CI pipeline would not catch any errors here (but not entirely sure), because in the GitHub windows runners the v7.4 LTS is installed